zazu/repo/commands.py

```diff
@@ -81,10 +87,62 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('--nohooks', is_flag=True, help='does not install git hooks in the repo')
 @click.pass_context
-def init(ctx):
+def init(ctx, nohooks):
     """Initialize repo directory structure"""
-    raise NotImplementedError
+    def _zazu_yaml(issue_tracker={}, stylers=[]):
+        """builds zazu.yaml file"""
+        if issue_tracker:
+            zazu_yaml_obj = issue_tracker
+        else:
+            zazu_yaml_obj = {}
+        if stylers:
+            zazu_yaml_obj['style'] = {key: {'options': [None]} for key in stylers}
+            click.echo('Reminder: please specify styler options in  zazu.yaml')
+        yaml.dump(zazu_yaml_obj, file('zazu.yaml', 'w'), default_flow_style=False)
+    # check for git repo in cwd
+    try:
+        repo = git.Repo(os.getcwd())
+    except git.InvalidGitRepositoryError:
+        event_time = time.gmtime()
+        default_name = time.mktime(event_time)
```

Comment:
I don't think that we should name the repo if the user doesn't, they should be required to name it.